src/dialogs/alwaysonbotDialogs/Common/FeedbackDialog.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    Choice, ChoiceFactory, ChoicePrompt, ComponentDialog, DialogTurnResult, PromptValidatorContext, TextPrompt,
+    WaterfallDialog,
+    WaterfallStepContext
+} from 'botbuilder-dialogs';
+
+import i18n from '../../locales/i18nconfig';
+
+
+const TEXT_PROMPT = 'textPrompt';
+const WATERFALL_DIALOG = 'waterfallDialog';
+const CHOISE_PROMPT = 'CHOISE_PROMPT'
+
+export const FeedBack_Dialog = 'FeedBack_Dialog';
```

Comment:
this name is not consistent with other file,

I do recommend change so we can keep them consistent
export const OAS_BENEFIT_DIALOG_STEP = 'OAS_BENEFIT_DIALOG_STEP'